scylla-cql/src/frame/value_tests.rs

```diff
@@ -490,46 +490,45 @@ fn cql_timestamp_serialization() {
 #[cfg(feature = "chrono")]
 #[test]
 fn naive_date_time_serialization() {
-    use chrono::NaiveDateTime;
-    let test_cases = [
+    use chrono::{DateTime, Utc};
+    let test_cases: [(DateTime<Utc>, [u8; 8]); 7] = [
```

Comment:
I think it is exactly as relevant as it was before.
Before we had NaiveDataTime array and we converted in to DateTime<Utc> before serialization (see .and_utc() call), now we operate on DateTime from the beggining.

Reply:
`I think it is exactly as relevant as it was before.` - I'd like to note that it does not imply that it is relevant, just that my change didn't make it worse

Reply:
I don't see `SerializeCql` impl for `NaiveDateTime`, so it would be hard for this name to be correct.

```
#[cfg(feature = "chrono")]
impl SerializeCql for NaiveDate {
    impl_serialize_via_writer!(|me, typ, writer| {
        exact_type_check!(typ, Date);
        <CqlDate as SerializeCql>::serialize(&(*me).into(), typ, writer)?
    });
}
#[cfg(feature = "chrono")]
impl SerializeCql for DateTime<Utc> {
    impl_serialize_via_writer!(|me, typ, writer| {
        exact_type_check!(typ, Timestamp);
        <CqlTimestamp as SerializeCql>::serialize(&(*me).into(), typ, writer)?
    });
}
#[cfg(feature = "chrono")]
impl SerializeCql for NaiveTime {
    impl_serialize_via_writer!(|me, typ, writer| {
        exact_type_check!(typ, Time);
        let cql_time = CqlTime::try_from(*me).map_err(|_: ValueOverflow| {
            mk_ser_err::<Self>(typ, BuiltinSerializationErrorKind::ValueOverflow)
        })?;
        <CqlTime as SerializeCql>::serialize(&cql_time, typ, writer)?
    });
}
```

Reply:
Let's correct the name then, as a bonus, in this PR.

Reply:
Done